src/sidebar/util/time.js

```diff
@@ -1,57 +1,13 @@
 'use strict';
 
-const minute = 60;
-const hour = minute * 60;
-
-function lessThanThirtySecondsAgo(date, now) {
```

Comment:
This and the following several functions that serve as tests have been converted into anonymous functions in the `BREAKPOINTS` object.